compose-builder/Makefile

```diff
@@ -458,8 +478,8 @@ build:
 	make compose ds-rest ds-virtual arm64
 	make compose ds-rest ds-virtual no-secty
 	make compose ds-rest ds-virtual no-secty arm64
-	make compose ds-rest ds-virtual no-secty ui
-	make compose ds-rest ds-virtual no-secty arm64 ui
+	make compose ds-rest ds-virtual asc-sample no-secty ui
+	make compose ds-rest ds-virtual asc-sample no-secty arm64 ui
```

Comment:
so this `asc-sample` is only for no-secty and ui?  there is no asc-sample in lines above like 478-480.

Reply:
Yep, this is so the UI has it to allow users to modify the configurable function pipeline. Not needed in any of the other versions.